projects/plugins/boost/app/assets/src/js/features/minify-meta/minify-meta.tsx

```diff
@@ -1,72 +1,92 @@
-	const [ isEditing, setIsEditing ] = useState( false );
+	const [ isExpanded, setIsExpanded ] = useState( false );
 
 	useEffect( () => {
 		setInputValue( values.join( ', ' ) );
-	}, [ values ] );
+	}, [ values, isExpanded ] );
 
 	function save() {
+		const type = datasyncKey === 'minify_js_excludes' ? 'js' : 'css';
+		recordBoostEvent( 'concatenate_' + type + '_exceptions_save_clicked', {} );
 		updateValues( inputValue );
-		setIsEditing( false );
+		setIsExpanded( false );
 	}
 
+	const toggleExpanded = ( newValue: boolean ) => {
+		const type = datasyncKey === 'minify_js_excludes' ? 'js' : 'css';
+		recordBoostEvent( 'concatenate_' + type + '_show_options_toggle', {
+			status: newValue ? 'open' : 'close',
+		} );
+		setIsExpanded( newValue );
+	};
+
 	const htmlId = `jb-minify-meta-${ datasyncKey }`;
 
 	return (
-		<div className={ styles[ 'minify-meta' ] } data-testid={ `meta-${ datasyncKey }` }>
-			{ isEditing ? (
-				<div className={ styles[ 'manage-excludes' ] }>
-					<label htmlFor={ htmlId }>{ inputLabel }</label>
-					<input
-						type="text"
-						value={ inputValue }
-						placeholder={ placeholder }
-						id={ htmlId }
-						onChange={ e => setInputValue( e.target.value ) }
-					/>
-					<div className={ styles[ 'buttons-container' ] }>
-						<button disabled={ values.join( ', ' ) === inputValue } onClick={ save }>
-							{ __( 'Save', 'jetpack-boost' ) }
-						</button>
-						<button onClick={ () => setIsEditing( false ) }>
-							{ __( 'Cancel', 'jetpack-boost' ) }
-						</button>
-					</div>
+		<div className={ styles.wrapper } data-testid={ `meta-${ datasyncKey }` }>
+			<div className={ styles[ 'minify-meta' ] }>
+				<div className={ styles.summary }>
+					{ values.length > 0 && (
+						<div className="successes">
+							{ sprintf(
+								/* Translators: %s refers to the list of excluded items. */
+								__( 'Except: %s', 'jetpack-boost' ),
+								values.join( ', ' )
+							) }
+						</div>
+					) }
 				</div>
-			) : (
-				<>
-					<div className={ styles.summary }>
-						{ values.length > 0 && (
-							<div className="successes">
-								{ sprintf(
-									/* Translators: %s refers to the list of excluded items. */
-									__( 'Except: %s', 'jetpack-boost' ),
-									values.join( ', ' )
-								) }
-							</div>
-						) }
-					</div>
 
-					<Button
-						variant="link"
-						size="small"
-						weight="regular"
-						className={ styles[ 'edit-button' ] }
-						onClick={ () => setIsEditing( true ) }
-						icon={ <Pencil /> }
-					>
-						{ buttonText }
-					</Button>
-				</>
+				<Button
+					variant="link"
+					size="small"
+					weight="regular"
+					className={ styles[ 'edit-button' ] }
+					onClick={ () => toggleExpanded( ! isExpanded ) }
+					icon={ isExpanded ? <ChevronUp /> : <ChevronDown /> }
+				>
+					{ isExpanded ? __( 'Hide', 'jetpack-boost' ) : buttonText }
+				</Button>
+			</div>
+			{ isExpanded && (
```

Comment:
Let's use collapsible meta as you suggested. It will help us keep the complex part of the interaction reusable.

Reply:
I removed the Hide module from the minify meta, and from the imageCDN quality settings to have a consistent codebase to start moving things to CollapsibleMeta